doc/installation-guide.rst

```diff
@@ -192,6 +192,7 @@ It is a a maven application:
 ``       $ mvn assembly:assembly``
 
 -  copy target/distribution/puppwrapper-dist to a desired location
+-  run ./generateselfsigned.sh on puppetwrapper-dist/bin
```

Comment:
maybe it is better to change to:

run <on_puppetwrapper_dist>/bin/generateselfsigned.sh?